tutorial/practicals/nemo/3_nemo_openmp/solutions/parallel_region_omp_trans.py

```diff
@@ -49,6 +49,15 @@
 from psyclone.transformations import OMPParallelLoopTrans, OMPLoopTrans, \
     OMPParallelTrans, TransformationError
```

Comment:
There are a couple of other scripts under `solutions` that need updating too.

Reply:
Done